homework2/firstTask.js

```diff
@@ -0,0 +1,24 @@
+// It returns time difference
+
+function durationBetweenDates(startDate = "16 Jan 1995", endDate = "24 Feb 2022", dimension = "seconds") {
+    let differenceInSeconds = Math.abs(Date.parse(endDate) - Date.parse(startDate))/1000;
+    let result;
+
+    switch (dimension) {
+        case "days":
+            result = Math.floor(differenceInSeconds/86400);
+            break;
+        case "hours":
+            result = Math.floor(differenceInSeconds/3600);
+            break;
+        case "minutes":
+            result = Math.floor(differenceInSeconds/60);
+            break;
+        default:
+            result = differenceInSeconds;
+    }
+    return result;
```

Comment:
```suggestion
    return `${result} ${dimension}`;
```
Все зроблено вірно, тільки тут забули додати dimension